products/opensuse/main.pm

```diff
@@ -605,7 +605,7 @@ sub load_x11tests() {
         loadtest "x11/nautilus" unless get_var("LIVECD");
         loadtest "x11/gnome_music";
         loadtest "x11/evolution" unless is_server;
-        if (!get_var("USBBOOT")) {
+        if (!get_var("USBBOOT") && !get_var("LIVECD")) {
```

Comment:
because you are now using that all over maybe it makes sense to define it as a `is_…` method with a good name and comment/description